.pre-commit-config.yaml

```diff
@@ -45,6 +45,7 @@ repos:
                 - types-requests
                 - types-toml
                 - types-click
+                - traitlets~=5.0
```

Comment:
pre-commit runs mypy from an isolated virtual environment so it doesn't have access to the installed dependencies, and thus we need to add them here manually. This is highly problematic, as we are essentially duplicating info from setup.cfg. In the future, we probably have to switch to an approach used in aiida-core, where mypy is installed as dev dependency of aiidalab, instead of being installed by pre-commit.